python/lsst/dax/apdb/tests/_apdb.py

```diff
@@ -658,6 +658,12 @@ def test_nometadata(self) -> None:
             self.assertTrue(metadata.empty())
             self.assertIsNone(metadata.get("meta"))
 
+        # Also check what happens when configured schema has metadata, but
+        # database is missing it.
+        apdb = make_apdb(config)
+        metadata = apdb.metadata
+        self.assertTrue(metadata.empty())
```

Comment:
Sorry, I don't understand this test. How does it create a scenario where the DB is missing the metadata table? I assume that, with the above rename, `config` is still the baseline config...

Reply:
`config` here is indeed a baseline config which has metadata table, but the database schema is created above with `config_nometa` which is missing metadata table (it is dropped from configuration by `update_schema_yaml()`).

Reply:
Ah, got it. Then please add a comment noting that the APDB was initialized previously. It's a bit too easy to miss that the steps aren't independent.